Sillycore/Infrastructure/AssemblyScanner.cs

```diff
@@ -37,7 +37,7 @@ private static void LoadAssembly(Assembly assembly)
         {
             Assemblies.Add(assembly.FullName, assembly);
 
-            AssemblyName[] assemblyNames = assembly.GetReferencedAssemblies();
+            AssemblyName[] assemblyNames = assembly.GetReferencedAssemblies().Where(x => !x.FullName.StartsWith("Microsoft")).ToArray();
```

Comment:
That's completely irrelevant.

Reply:
You are right, but we add this to avoid some strange exception, assembly not found or something.